service/s3/s3manager/download.go

```diff
@@ -402,18 +407,31 @@ func (d *downloader) downloadChunk(chunk dlchunk) error {
 	// Get the next byte range of data
 	in.Range = aws.String(chunk.ByteRange())
 
+	var (
+		w       io.Writer
+		cleanup func()
+	)
+
 	var n int64
 	var err error
```

Comment:
If grouping `var` should the `n` and `err` be grouped as well? or alternative `w`and `cleanup` should be ungrouped.